### ISSUE TITLE
Why use freeze_bn during the training?

### ISSUE THREAD

**Issue by Wallace-222** [facebookresearch/DomainBed#9]
Thanks a lot for your wonderful job! 

Would you tell me why you are using freeze_bn during the training phase?

**Comment by lopezpaz** [comment]
As described in Appendix D from [our manuscript](https://arxiv.org/abs/2007.01434):

> Observing that batch normalization interferes with domain generalization
> algorithms (as different minibatches follow different distributions), we freeze all batch normalization
> layers before fine-tuning.
